controller/RestTask.php

```diff
@@ -38,6 +39,7 @@ class RestTask extends \tao_actions_CommonModule
 
     /**
      * @inheritdoc
+     * @security("hidden")
```

Comment:
is `hidden` correct, or should this be `hide`?

Reply:
of course `hide`,
good catch, thank you!